### DISCUSSION TITLE
Video Player issue

### DISCUSSION THREAD

**Discussion by Shogun-Z** [aniyomiorg/aniyomi#833] [General]
Totally New Over here , so idk what I am doing but Seriously love this app on phone ..

But Got some Major issue on my Firstick tv 

When you are watching anime on Firestick, You can't pause it.

You have to go back & Select the Episode over again to pause / Play / Forward / Backward it ! 

Can't choose External Player / External Downloader on FireStick too , It'll just Force Close if you choose External apps .

**Comment by kpowers1028** [comment]
If you choose to use an external player that will rectify that issue.

**Reply by kpowers1028** [reply]
My bad I didn't catch the last part. But I am currently running it on a Fire Cube with an external player that I sideloaded. MX player is the player I am using.

**Reply by Shogun-Z** [reply]
Yeah, I tried using Externally Player Ex VLC & MX Player .

After I select any one of em, Aniyomi Just Force Closes & Send me On home page

**Reply by Shogun-Z** [reply]
How ?? Did you side loaded MX player ? Or is it from the Firestick Store. 

Because mine Aniyomi Forces closes like crazy ,if I set Always use External Player

**Reply by kpowers1028** [reply]
Also make sure you have the switch flipped that says always use external player.

**Reply by Shogun-Z** [reply]
Okay, Which Aniyomi Version are you using ?? 

Stable ??  (which one ?? Amrv7 , 8 or global )
Preview ?? 
Beta ??

**Reply by kpowers1028** [reply]
Yes MX player is a little hard to find. Sideloading it is the only way I am aware of. Techdocuk I think is one place. But remember that your doing this at your own risk.

**Reply by kpowers1028** [reply]
Stable 0.12.3.8

**Reply by Shogun-Z** [reply]
Okay downloaded MX player 1.10.44 from the website you said ,

What about Aniyomi Version ??

**Reply by kpowers1028** [reply]
It's a fantastic app my son and I watch anime together on it. I wish you the best of luck in getting it working better for yourself.

**Reply by Shogun-Z** [reply]
One more thing , can you please tell me your MX player Version ?? Which you are using ?m

**Reply by kpowers1028** [reply]
Stable 0.12.3.8

**Reply by kpowers1028** [reply]
Stick with the one you have. This one I took I big risk using.

**Reply by kpowers1028** [reply]
Any luck?

**Reply by Shogun-Z** [reply]
Yesh, Thanks Mate!! Worked for me after some tweaks

**Reply by kpowers1028** [reply]
Awesome glad it worked out. No problem! Glad I could assist.
